src/lowering/desugar.ml

```diff
@@ -258,6 +263,28 @@ and exp' at note = function
       { it = I.LetD ({it = I.WildP; at = e.at; note = T.Any}, exp e);
         at = e.at; note = ()}], (unitE ()))
 
+and parenthetical = function
+  | None -> []
+  | Some par ->
+    (* fishing for relevant attributes in the parenthetical based on its static type *)
+    let cycles, clean_cycles =
+      if T.(sub par.note.note_typ (Obj (Object, [T.cycles_fld])))
+      then [fun parV -> dotE parV T.cycles_lab T.nat |> assignVarE "@cycles" |> expD], []
+      else [], [natE Mo_values.Numerics.Nat.zero |> assignVarE "@cycles" |> expD] in
```

Comment:
```suggestion
      else [], [] in
```
If you did this, wouldn't the old mechanism still work if you only apply at timeout but don't mention cycles?
Then the Changelog doesn't have to say the approaches can't be mixed.

Reply:
As you say, a `Cycles.add` would continue to work. Ah, you are worried about codebase migration, where folks want to add `(with timeout) but are hesitant to check for the presence of `Cycles.add` in front of the call?

Reply:
yep

Reply:
Hmm, I am not married to the idea, but I think we should deprecate the old way sooner than later. I'll make this change then.